lms/validation/_base.py

```diff
@@ -1,12 +1,25 @@
 """Base classes for validation schemas."""
 import marshmallow
-from marshmallow import pre_load
+from marshmallow import pre_load, fields
 from pyramid.httpexceptions import HTTPUnsupportedMediaType
 from webargs import pyramidparser
 
 from lms.services import ExternalRequestError
 from lms.validation._exceptions import ValidationError
 
+from marshmallow.utils import get_value
+
+
+class Reach(fields.Field):
```

Comment:
Marshmallow makes this a bit difficult.

It looks like you could use Pluck or Nested or some other combination but you have to do something like this (from: https://github.com/marshmallow-code/marshmallow/issues/1315#issuecomment-514266136)